packages/ui-react/src/containers/Layout/Layout.tsx

```diff
@@ -210,10 +214,16 @@ const Layout = () => {
         )}
       </div>
       <Sidebar isOpen={sideBarOpen} onClose={() => setSideBarOpen(false)}>
-        <MenuButton label={t('home')} to="/" tabIndex={sideBarOpen ? 0 : -1} />
-        {menu.map((item) => (
-          <MenuButton key={item.contentId} label={item.label} to={playlistURL(item.contentId)} tabIndex={sideBarOpen ? 0 : -1} />
-        ))}
+        <ul>
+          <li>
+            <MenuButton label={t('home')} to="/" tabIndex={sideBarOpen ? 0 : -1} />
+          </li>
+          {menu.map((item) => (
+            <li key={item.contentId}>
+              <MenuButton label={item.label} to={playlistURL(item.contentId)} tabIndex={sideBarOpen ? 0 : -1} />
+            </li>
+          ))}
+        </ul>
         <hr className={styles.divider} />
```

Comment:
Don't we want to replace this one with your solution as well?

Reply:
I intentionally skipped  this `<hr>` due to time and I do not consider it a requirement (or something we could fail upon during an audit).  But since you mention it, I took the effort and fixed it ;)